src/fernet/core.clj

```diff
@@ -41,14 +43,17 @@
                 expected-signature))
       (invalid-token))))
 
-(defn aes128cbc [mode key iv message]
-  (.doFinal (doto (Cipher/getInstance "AES/CBC/PKCS7Padding")
-              (.init
-                (mode {:encrypt Cipher/ENCRYPT_MODE
-                       :decrypt Cipher/DECRYPT_MODE})
-                (new SecretKeySpec key "AES")
-                (new IvParameterSpec iv)))
-            message))
+(defn aes128cbc
```

Comment:
This is an insecure (in the IND-CCA2 sense) cryptographic primitive, so it should be private. (It is insecure because it is unauthenticated, meaning that it provides neither authenticity nor integrity in an IND-CCA2 scenario.)